lib/Extension/LanguageServerRename/Adapter/ReferenceFinder/ClassMover/ClassRenamer.php

```diff
@@ -81,7 +85,7 @@ public function rename(TextDocument $textDocument, ByteOffset $offset, string $n
 
         $originalName = $this->getFullName($node);
         $newName = $this->createNewName($originalName, $newName);
-        $oldUri = $this->nameToUriConverter->convert($originalName);
+        $oldUri = $this->reflector->reflectClass($originalName->getFullyQualifiedNameText())->sourceCode()->uri();
```

Comment:
To be consistent there should be an adapter in `Adapter/WorseReflection/WorseNameToUriConverter` which implements `Model\NameToUriConverter`. You can then inject that instead.